### PR TITLE
Jenchen/main alert subscription

### PR DESCRIPTION
**Pull request by Jennifer-chen-rh**
Some fixes from previous code reviews

### CODE REVIEW

File: internal/service/alarm_subscription_handler.go [hunk 1]
```diff
@@ -300,11 +286,12 @@ func (h *alarmSubscriptionHandler) getSubcriptionId() (subId string) {
 // add it for now for the test purpose
 func (h *alarmSubscriptionHandler) encodeSubId(ctx context.Context,
 	subId string, input data.Object) (output data.Object, err error) {
-	//get cluster name, subscriptions
+	//get customer name, subscriptions
```
Comment (irinamihai) [inline internal/service/alarm_subscription_handler.go:289]: Maybe you meant "consumer" instead of "customer"?

Comment (Jennifer-chen-rh) [reply internal/service/alarm_subscription_handler.go]: Yes. changed thanks

**Pull request merged**